src/app/page.tsx

```diff
@@ -1,18 +1,10 @@
 import Card from './components/Card';
 
-function repeat(cards: JSX.Element[], repetition: number = 10): JSX.Element[] {
-  const items: JSX.Element[] = [];
-  for (let i: number = 0; i < repetition; ++i)
-    items.push(cards[Math.floor(Math.random() * cards.length)]);
-
-  return items;
-}
-
 export default function Home(): JSX.Element {
   const cards: JSX.Element[] = [
-    Card({ cardType: 0 }),
-    Card({ cardType: 1 }),
-    Card({ cardType: 2 }),
+    <Card key={Math.floor(Math.random())} cardType={0} />,
```

Comment:
`Math.floor(Math.random())`
wont this recalculate every rerender?

Reply:
when use Card({ ... }) the eslint not raise error about the key prop, but using component tag directly it need us to use the key prop

also later probably the key value will retrieved from record's id so wont recalculate on each render, should we use the Card() again or continue to use component tag? based on convention array values of JSX.Element[] type

Reply:
after see the code again I realize this only for mockup data, so I added todo comment on https://github.com/RR-Ardhika/murojaah/pull/3/commits/494ab4d2197299e82a6bfe2ae85cf1a721b31d52